mkdocs.yml

```diff
@@ -15,13 +15,13 @@ edit_uri: edit/main/docs/
 
 # Plugins
 plugins:
-    - search
-    - git-revision-date-localized:
-        type: timeago
-        enable_creation_date: true
-    - git-committers:
-        repository: JanssenProject/jans
-        token: !ENV GH_TOKEN
+#    - search
+#    - git-revision-date-localized:
+#        type: timeago
+#        enable_creation_date: true
+#    - git-committers:
+#        repository: JanssenProject/jans
+#        token: !ENV GH_TOKEN
```

Comment:
did you mean to push this ? We want to (add committers- this needs more work) and keep last revision to the pages

Reply:
No, I didn't mean to push this. I'll remove it. 
I commented it to make the local build faster and the plan was to remove it before committing.